apply-pr.js

```diff
@@ -149,6 +149,10 @@ function resolveNames() {
       exitWithMsg(err);
     }
 
+    if (!name.email) {
+      exitWithMsg(util.format('Email not set for %s', name.name));
```

Comment:
maybe we can add something like --no-email to the commandline arguments that people can still use git apply-pr and can fix the commit message later manually before pushing?

@misterdjules what do you think?